DNN Platform/Library/Services/UserProfile/UserProfilePicHandler.cs

```diff
@@ -163,16 +163,7 @@ public void ProcessRequest(HttpContext context)
             context.Response.Cache.SetExpires(DateTime.Now.AddMinutes(1));
             context.Response.Cache.SetMaxAge(new TimeSpan(0, 1, 0));
             context.Response.AddHeader("Last-Modified", DateTime.Now.ToString("r"));
-
-	        try
-	        {
-				context.Response.End();
-	        }
-			catch (ThreadAbortException)//if ThreadAbortException will shown, should catch it and do nothing.
-			{
-
-			}
-            
+            context.ApplicationInstance.CompleteRequest();
```

Comment:
I don't really understand the full repercussions of this change, but this is my only outstanding question on the full PR, so if anyone else can sign off on this, the rest gets my :+1: